src/test/regress/expected/isolation_ref_update_delete_upsert_vs_all_on_mx.out

```diff
@@ -227,22 +227,6 @@ run_commands_on_session_level_connection_to_node
 
 (1 row)
 
-step s2-start-session-level-connection:
-        SELECT start_session_level_connection_to_node('localhost', 57638);
```

Comment:
is it ok that we're removing this?

Reply:
I did not check but the removal of those lines comes from the reversal commit itself which is approved before.